react-components/src/metadata/admin/EditMetadataTemplate.js

```diff
@@ -85,8 +63,8 @@ class EditMetadataTemplate extends Component {
             classes,
             intl,
             open,
-            // from redux-form
-            handleSubmit, pristine, submitting, error, change,
```

Comment:
🎉 We no longer have to pass a `change` function from the top-level form down to each attribute and enum editor, since Formik provides each field access to top-level form functions like `setFieldValue `.